Rakefile

```diff
@@ -1,10 +1,12 @@
 require 'rake/testtask'
 require 'bundler/gem_tasks'
+require 'ci/reporter/rake/minitest'
 
 Rake::TestTask.new do |t|
   t.libs << "lib"
   t.test_files = Dir.glob('test/**/*_test.rb')
   t.verbose = true
+  t.warning = false
```

Comment:
I turned the warnings off because upgrading Rake caused all sorts of warnings to start popping up, many come from libraries. Example:

```
/home/vagrant/hammer-cli-katello/.vendor/ruby/2.7.0/gems/clamp-1.3.2/lib/clamp/attribute/instance.rb:23: warning: instance variable @subcommand_arguments not initialize
```

```
/home/vagrant/hammer-cli-katello/.vendor/ruby/3.0.0/gems/fast_gettext-2.3.0/lib/fast_gettext/vendor/string.rb:70: warning: too many arguments for format string
```

Reply:
That's fine, rather have green tests than ruby warnings